<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>261</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>217</width>
    <height>261</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>217</width>
    <height>261</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CSV File Compressor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ChooseFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDoParallel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Do Parallel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ActButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Act</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMethod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
